python/Ganga/GPIDev/Credentials/GridProxy.py

```diff
@@ -206,12 +204,10 @@ def chooseCommandSet(self):
         Return value: None
         """
         from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList
-        if "ICommandSet" == self.command._name or isType(self.command, GangaList):
-            if self.voms:
-                self.command = self.vomsCommand
-            else:
-                self.command = self.gridCommand
-        return None
+        if self.voms:
```

Comment:
@milliams Isn't it possible to have a .gangarc which is used in the tests for GangaDirac etc such that this is the config known to work on this machine?
It should be possible to add an argument to the ```GangaUnitTest``` to use the config file from wherever we put it, so we could potentially just wget a file from github at the start of the test and tell GangaUnitTest to use that config for that test surely?